charts/psmdb-operator/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: "1.11.0"
+appVersion: "1.12.0"
 description: A Helm chart for Deploying the Percona Kubernetes Operator for Percona Server for MongoDB
 name: psmdb-operator
 home: https://www.percona.com/doc/kubernetes-operator-for-psmongodb/kubernetes.html
-version: 1.11.0
+version: 1.12.0
 maintainers:
```

Comment:
do we need this 
`- name: paulczar
    email: username.taken@gmail.com`

Reply:
Thanks, it is updated now!